datafusion/core/src/physical_optimizer/enforce_distribution.rs

```diff
@@ -2056,23 +2056,33 @@ pub(crate) mod tests {
     }
 
     /// Runs the repartition optimizer and asserts the plan against the expected
+    /// Arguments
```

Comment:
I found it very hard to read some of the tests as a line like this

```
        assert_optimized!(expected, physical_plan, false, true);
```

As I had to go back to the macro definitions several times to remember what the `false` and `true` parameters meant